python-sdk/tests/files/locations/test_location_base.py

```diff
@@ -108,25 +109,20 @@ class SomethingElseLocation:  # skipcq: PY-D0002
 @pytest.mark.parametrize(
     "loc_1,loc_2,equality",
     [
-        (LocalLocation("/tmp/file_a.csv"), LocalLocation("/tmp/file_a.csv"), True),
+        (GCSLocation("gs://tmp/file_a.csv"), GCSLocation("gs://tmp/file_a.csv"), True),
         (
-            LocalLocation("/tmp/file_a.csv", conn_id="test"),
-            LocalLocation("/tmp/file_a.csv", conn_id="test"),
+            GCSLocation("gs://tmp/file_a.csv", conn_id="google_cloud_default"),
```

Comment:
@sunank200 Why did we modify this test case?

Reply:
The `conn_id` is validated as part of BaseFileLocation call